.gitignore

```diff
@@ -4,3 +4,4 @@ docs
 node_modules
 peaks.js
 peaks.js.map
+.idea/
```

Comment:
This seems to be specific to your development environment. Please use your [global .gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) instead.

Reply:
TIL, thanks